.travis.yml

```diff
@@ -11,19 +11,20 @@ matrix:
   allow_failures:
     - env: TOXENV=isort
 before_cache:
-  - rm -f $HOME/.cache/pip/log/debug.log
+  - pyclean $HOME/virtualenv/python2.7.9/lib
 cache:
   directories:
     - pootle/static/js/node_modules
     - pootle/assets
-    - $HOME/.cache/pip
+    - $HOME/virtualenv/python2.7.9/lib
+    - $HOME/virtualenv/python2.7/bin
+
 before_install:
   # Error as early as possible if we can't compile
   - python -m compileall -q -f -x ".*[.]tox.*" .
 install:
   - pip install --upgrade "pip>=8.0" # FIXME remove once Travis support pip8 by default
-  - pip install --upgrade "tox>=2.3"
-  - pip install
+  - pip install --upgrade
```

Comment:
I think you want skip upgrade, since that would requires pings to pypi which would happen anyway if the local requirement didn't meet ``requirements/*``.  Having said that removing ``--upgrade`` from pip and tox upgrade didn't show any real speedup on my tests.